### PR TITLE
[SD-3859] - Saved search list is sorted

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: client/app/scripts/superdesk-monitoring/aggregate.js [hunk 1]
```diff
@@ -489,7 +489,7 @@
                     if (scope.privateSavedSearches.length > 0) {
                         scope.privateSavedSearches.length = 0;
                     }
-                    _.each(scope.searches, function(item) {
+                    _.each(_.sortBy(scope.searches, 'name'), function(item) {
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-monitoring/aggregate.js]: We are using `sortByName` filter for this kind of functionality. Can we use the same to be consistent with other lists also?

**Comment by mugurrus** [comment]
Some e2e tests need to be updated because of the sort by name

**Comment by mdhaman** [comment]
:+1:

**Comment by marwoodandrew** [comment]
:+1:

**Comment by sivakuna-aap** [comment]
:+1:

**Pull request merged**